src/main/scala/org/cs1331/gitsubmitter/GitSubmitterPlugin.scala

```diff
@@ -0,0 +1,30 @@
+package org.cs1331.gitsubmitter
+
+import java.time.LocalDateTime
+import sbt._
+import Keys._
+
+object GitSubmitterPlugin extends AutoPlugin {
+
+  override lazy val projectSettings = Seq(commands += submitCommand)
+
+  lazy val submitCommand = Command.command("submit") { (state: State) =>
+    val prismId = readLine("Enter your prism id: ")
+    val repoSuffix = s"-$prismId"
+    val submission = Application.authenticateAndCreate(repoSuffix)
+    submission.createRepo()
+    submission.addCollab("cs1331-fall2015")
+    val commitMsg = s"Solution progress at ${LocalDateTime.now}"
+
+    // This line fails. I don't know how to use StudentSubmission.pushFile ...
+    // it's not like git-add, which you can just hand a directory. Maybe
+    // we need to list all the files in src/main/java and 'push' them one by one?
+    // That seems inelegant.
+    submission.pushFile(new File("src/main/java/"), commitMsg)
```

Comment:
Sorry for the miscommunication on my part. Obviously it'd be better to have an actual local repo, but that introduces huge complications with authentication which the Github API solves nicely. At some point, we can move over to use JGit (and thus a local repo), but I'm afraid we just don't have the time right now.